implementation/helper/boost/asio/detail/impl/socket_ops_ext_local.ipp

```diff
@@ -72,15 +98,29 @@ signed_size_type recv(socket_type s, buf* bufs, size_t count,
 		 cmsg != NULL;
 	     cmsg = CMSG_NXTHDR(&msg, cmsg))
 	{
+#ifdef __QNX__
+	  if (cmsg->cmsg_level != SOL_SOCKET || cmsg->cmsg_type != SCM_CREDS
+	      || cmsg->cmsg_len != CMSG_LEN(sizeof(UCRED_T)))
+	    continue;
+#else
 	  if (cmsg->cmsg_level != SOL_SOCKET || cmsg->cmsg_type != SCM_CREDENTIALS
-	      || cmsg->cmsg_len != CMSG_LEN(sizeof(struct ucred)))
+	      || cmsg->cmsg_len != CMSG_LEN(sizeof(UCRED_T)))
```

Comment:
Why this change? UCRED_T seems to only be defined within __QNX__ environments.

Reply:
This is a mistake. Thanks for pointing it out.